coderd/database/dbauthz/interface.go

```diff
@@ -0,0 +1,11 @@
+package dbauthz
+
+import "github.com/coder/coder/coderd/database"
+
+// AuthzStore is the interface for the Authz querier. It will track closely
+// to database.Store, but not 1:1 as not all database.Store functions will be
+// exposed.
+type AuthzStore interface {
```

Comment:
Because this package is called `dbauthz`, we can just name this `Store`. Then it fits nicely with `database.Store` too!

I'd toss this file in `dbauthz.go` too. It's weird for the primary interface to be in a separate file.